WeakAuras/Transmission.lua

```diff
@@ -156,12 +157,13 @@ local function filterFunc(_, event, msg, player, l, cs, t, flag, channelId, ...)
       newMsg = newMsg..remaining:sub(1, start-1);
       newMsg = newMsg.."|Hgarrmission:weakauras|h|cFF8800FF["..characterName.." |r|cFF8800FF- "..displayName.."]|h|r";
       remaining = remaining:sub(finish + 1);
+      anyLinkFound = true
     else
       newMsg = newMsg .. remaining
```

Comment:
Previously, this branch (which should be immediately taken in all cases where we get a whisper that doesn't have a WA link in it) would not append `remaining` onto `newMsg`. Thus, the loop would end with `newMsg` being just the empty string, and we would do nothing to the whisper. Now, it is never the case that `newMsg` is an empty string (unless the message itself is an empty string, i suppose), so we would always enter the block below, even if it's a perfectly innocent whisper.